crates/astria-composer/src/executor/mod.rs

```diff
@@ -553,6 +590,22 @@ impl Future for SubmitFut {
                     Ok(rsp) => {
                         let tendermint::abci::Code::Err(code) = rsp.code else {
                             info!("sequencer responded with ok; submission successful");
+
+                            metrics::gauge!(crate::metrics_init::CURRENT_NONCE)
+                                .set(*this.nonce + 1);
+
+                            // allow: the number of bytes in a bundle will be large enough to not
+                            // cause a precision loss
+                            #[allow(clippy::cast_precision_loss)]
+                            metrics::histogram!(crate::metrics_init::BYTES_PER_SUBMISSION)
+                                .record(this.bundle.get_size() as f64);
+
+                            // allow: a bundle will have at least 1 transaction which will not cause
```

Comment:
The precision loss is not important for small `usize`, but for large. But the main reason for this is:
```suggestion
                            // allow: precision loss is unlikely (values too small) but also unimportant in histograms
```